perllib/Dataset/UK/Stats19/Schema/Result/Accident.pm

```diff
@@ -272,4 +272,30 @@ sub grouped_participants {
     );
 }
 
+sub most_significant_participant {
```

Comment:
Feels a bit odd that category could be 'horse' (as grouped_participants has all vehicles as 2, horse as 3) but this would be 'hgv' - though I imagine that's pretty unlikely!

Reply:
I am not sure why horse is more important than vehicles there as I'd expect for most road safety purposes it's the thing most likely to do damage that's the most important vehicle. Given that I might reduce the horse rating here.

Reply:
Yeah, did seem odd, wondered if joke/always wanting to highlight if a horse was involved. Could reduce horse other place, just thought probably wanted to match it in some way, but guess doesn't really matter.